src/FileSystem.gren

```diff
@@ -102,6 +104,21 @@ type FileHandle readAccess writeAccess
     = FileHandle
 
 
+{-| Represents items in a directory.
+
+Variants for each type of file, holding a `String` with the filename.
```

Comment:
Could it be confusing to refer to 'file' here, when `File` is one of the variants? I know that it's technically correct for Unix-systems. What about:

```
Variants for each type of entry, with a `String` representing the relative path.
```

Or is that just more confusing? 😅

Reply:
Agree with this and the above comment. Much better!